src/chi-vue/src/components/data-table-filters/AdvancedFilters.tsx

```diff
@@ -221,67 +227,66 @@ export default class AdvancedFilters extends Vue {
         }
       });
-    const advancedFiltersButton = (
-      <button
-        id={this._advancedFilterButtonId}
-        onclick={() => this._toggleAdvancedFiltersPopover()}
-        class={`${BUTTON_CLASSES.BUTTON} ${PORTAL_CLASS} ${BUTTON_CLASSES.ICON_BUTTON} ${BUTTON_CLASSES.PRIMARY} ${BUTTON_CLASSES.FLAT}`}>
-        <div class={BUTTON_CLASSES.CONTENT}>
-          <i class={`${ICON_CLASS} icon-filter`} />
-        </div>
-      </button>
-    );
-    const advancedFiltersPopover = (
-      <chi-popover
-        ref="popover"
-        position="bottom"
-        reference={`#${this._advancedFilterButtonId}`}
-        title="Filters"
-        portal
-        drag
-        closable>
-        <div class={`${ACCORDION_CLASSES.ACCORDION} -sm`} ref="advancedFiltersAccordion">
-          {advancedFilters}
-        </div>
-        <div class={`advanced-filters__actions -mt--2`}>
-          <button
-            onclick={() => this._applyAdvancedFiltersChange()}
-            class={`${BUTTON_CLASSES.BUTTON} ${BUTTON_CLASSES.PRIMARY}`}
-            ref="advancedFiltersApplyButton"
-            disabled={
-              this.advancedFiltersData &&
-              compareFilters &&
-              compareFilters(this.$props.filtersData, this.advancedFiltersData)
-            }>
-            APPLY
-          </button>
-          <button onclick={() => this._toggleAdvancedFiltersPopover()} class={`${BUTTON_CLASSES.BUTTON} -ml--2`}>
-            CANCEL
-          </button>
-          <button
-            class={`
-              ${BUTTON_CLASSES.BUTTON}
-              ${PORTAL_CLASS} ${BUTTON_CLASSES.ICON_BUTTON}
-              ${BUTTON_CLASSES.PRIMARY}
-              ${BUTTON_CLASSES.FLAT} -ml--2 -bl--1`}
-            aria-label="Reset advanced filters"
-            onclick={() => this._resetAdvancedFilters()}>
-            <div class={BUTTON_CLASSES.CONTENT}>
-              <i class={`${ICON_CLASS} icon-reload`} />
-            </div>
-          </button>
-        </div>
-      </chi-popover>
-    );
 
-    return (
+    const advancedFiltersPopover = (
       <div>
-        {advancedFiltersButton}
-        {advancedFiltersPopover}
+        {!this.mobile && [
+          <div class={`${ACCORDION_CLASSES.ACCORDION} -sm`} ref="advancedFiltersAccordion">
+            {advancedFilters}
+          </div>,
+          <div class={`advanced-filters__actions -mt--2`}>
+            <button
+              onclick={() => this._applyAdvancedFiltersChange()}
+              class={`${BUTTON_CLASSES.BUTTON} ${BUTTON_CLASSES.PRIMARY}`}
+              ref="advancedFiltersApplyButton"
+              disabled={
+                this.filterElementValueLive && compareFilters(this.filterElementValue, this.filterElementValueLive)
+              }>
+              APPLY
+            </button>
+            <button onclick={() => this._toggleAdvancedFiltersPopover()} class={`${BUTTON_CLASSES.BUTTON} -ml--2`}>
```

Comment:
Not sure why but the cancel button does not work for me

https://user-images.githubusercontent.com/25136131/125335761-37051900-e34d-11eb-9bd2-e9ed026b55cf.mov